### ISSUE TITLE
Support auto-translate tools

### ISSUE THREAD

**Issue by elwinschmitz** [rodekruis/helpful-information#601]
The content of the site should be accessible via an auto-translate tool like Google Translate, or browser-extensions/features. 

## To Do
- [x] Add `lang`-attribute to HTML + setting in `ENV`-variable or sheet-field
- [x] Adjust/fix CSS to accommodate top-floating-toolbar/body-margin-top
- [x] #638
- [x] #639 

---

## Example: Google Translate
- Using the Google Translate website, using a URL to translate:  
  This results in access via a 'proxy' via the URL: https://helpfulinformation-redcross-nl.translate.goog/  
  So the Google Sheets API-key should allow this.  
  See: https://support.google.com/translate/answer/2534559?p=Website_Translation&hl=en
- Chrome in-browser translation?

## Example: Bing/Edge
- Edge in-browser translation?

## Example: Browser-extensions?
- 

Related #151

**Comment by elwinschmitz** [comment]
Added the `*.translate.goog`-domain to the API-key rules. This makes it work. ✅

[ ] Should be added to the deployment-guide/set-up instructions.

**Comment by elwinschmitz** [comment]
Use direct link/switch:

    https://translate.google.com/translate?u=https://example.org/specific-url

Set "Source Language" (`sl=`) and "Target Language" (`tl=`)

    https://translate.google.com/translate?sl=   en   &tl=   nl   &u=https://example.org/en/english-specific-url

**Comment by elwinschmitz** [comment]
Use +/- 3 presets: Nederlands, French, Arabic, ...
